### PR TITLE
[KNI] gh actions: update for branch 4.14

### PR DESCRIPTION
**Pull request by ffromani**
Fix automation for branch 4.14

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: CI Go
 
 on:
   push:
-    branches: [ master, release-4.12 ]
+    branches: [ master, release-4.12, release-4.14 ]
```
Comment (Tal-or) [inline .github/workflows/ci.yaml:5]: What about 4.13?

Comment (ffromani) [reply .github/workflows/ci.yaml:5]: is not a long term support. We cover only LTS releases here

**Review by swatisehgal — APPROVED** [review]
/lgtm

**Pull request merged**